strings/french.strings

```diff
@@ -84,9 +84,6 @@
 /* MISSING TRANSLATION - views/containers/contributions/GroupPledgesGraph.vue */
 "{span_}Group{_span} goal" = "{span_}Group{_span} goal";
 
-/* MISSING TRANSLATION - views/containers/dashboard/GroupMembersAllModal.vue, views/containers/dashboard/GroupMembersMenu.vue, views/pages/DesignSystem.vue */
-"Remove member" = "Remove member";
-
```

Comment:
Hmm, I'm not sure why these `.strings` files have been modified... could it be an outdated commit? Can you try merging `master` and see if that fixes this?

Reply:
The file don't exist anymore

Reply:
I don't understand what causes these changes. It happens the same too me and I never touched the files...

@taoeffect maybe you should create a PR to explicitly say how to update the strings and it should be a quick and straightforward way without the need to manually install other stuff.